extern/sector-storage/worker_local.go

```diff
@@ -1,16 +1,23 @@
 package sectorstorage
 
 import (
+	"bufio"
+	"bytes"
 	"context"
 	"encoding/json"
 	"io"
+	"math"
 	"os"
+	"path/filepath"
 	"reflect"
 	"runtime"
+	"strconv"
 	"sync"
 	"sync/atomic"
 	"time"
 
+	"github.com/containerd/cgroups"
+	cgroupv2 "github.com/containerd/cgroups/v2"
```

Comment:
We'll need to put cgroup stuff behind some build flag so that OSX build still works